scripts/insert_into_appointments.sql

```diff
@@ -1 +1,9 @@
 -- insert appointments creating connections between doctors and patients
+
+INSERT INTO appointments(date, time, comment, doctor_id, patient_id) VALUES
+  ('2016-08-12', '10:00:00', null, 4, 8),
+  ('2016-08-17', '13:00:00', null, 6, 5),
+  ('2016-08-12', '10:00:00', null, 6, 8),
+  ('2016-08-17', '13:00:00', null, 4, 7)
+
+;
```

Comment:
Hmm, I don't think that would work but I'm no SQL expert. Here's some working SQL:

```sql
-- insert appointments creating connections between doctors and patients
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT DATE '2017-08-12', TIME '1000',
    (SELECT id FROM doctors WHERE family_name = 'Hilton'),
    (SELECT id FROM patients WHERE family_name = 'Wilcox')
  UNION
  SELECT '2017-08-17', '1300',
    (SELECT id FROM doctors WHERE family_name = 'Hubbard'),
    (SELECT id FROM patients WHERE family_name = 'Talley')
  UNION
  SELECT '2017-08-17', '1000',
    (SELECT id FROM doctors WHERE family_name = 'Hubbard'),
    (SELECT id FROM patients WHERE family_name = 'Wilcox')
  UNION
  SELECT '2017-08-17', '1300',
   (SELECT id FROM doctors WHERE family_name = 'Hilton'),
   (SELECT id FROM patients WHERE family_name = 'Hodge')
;
```